client/src/pages/Login/Login.tsx

```diff
@@ -55,6 +54,7 @@ export default function Login(): JSX.Element {
               </Grid>
             </Grid>
             <LoginForm handleSubmit={handleSubmit} />
+            <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
```

Comment:
```suggestion
            <AuthFooter linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
```
Nit: Let's rename this component as we're using it as a footer for the auth forms

Reply:
Done the changes